run-checks

```diff
@@ -334,6 +334,35 @@ if [ "$STATIC" = 1 ]; then
       echo "$got"
       exit 1
     fi
+
+    echo "Checking installation of golangci-lint"
+    GOLANGCI_INSTALL_DIR="/tmp"
+    GOLANGCI_LINT="$GOLANGCI_INSTALL_DIR/golangci-lint"
+    # Ensure GOLANGCI_VERSION matches the golangci-lint version in
+    # .github/workflows/test.yaml (without leading v)
+    GOLANGCI_VERSION="1.51.2"
+    if ! command -v $GOLANGCI_LINT > /dev/null || ! $GOLANGCI_LINT --version | grep -q $GOLANGCI_VERSION ; then
```

Comment:
is not enough with 
! command -v $GOLANGCI_LINT > /dev/null ?

Reply:
I wanted to match the exact version used in the github workflow to avoid linting errors if/when golangci-lint updates and becomes more strict. But it's true, the latter check will fail too if golangci-lint isn't installed, so maybe just `! $GOLANGCI_LINT --version | grep -q $GOLANG_VERSION` is required.